ssg/build_remediations.py

```diff
@@ -258,10 +246,6 @@ class BashRemediation(Remediation):
     def __init__(self, file_path):
         super(BashRemediation, self).__init__(file_path, "bash")
 
-    def load_associated_rule(self, resolved_rules_dir, rule_id):
-        # No point in loading rule for this remediation type as of now
-        pass
-
 
 class AnsibleRemediation(Remediation):
```

Comment:
Add `isfile` check also into the `from_snippet_and_rule` method of this class.

Reply:
OK, I will do that.

Reply:
done